LevelUp.Pos.ProposedOrders.Tests/SplitTenderTests.cs

```diff
@@ -287,8 +248,69 @@ public void SplitTenderExample_LevelUp_Then_Cash_With_Adjusted_Exemption_And_Adj
                     exemptionAmount: 400);
 
             AdjustedCheckValues completedOrderValues = ProposedOrderCalculator.CalculateCompleteOrderValues(
-                totalOutstandingAmount,
-                taxAmountDue,
+                check.TotalOutstandingAmount,
+                check.TotalTaxAmount,
+                exemptionAmount,
+                spendAmount,
+                availableDiscountAmount);
+
+            completedOrderValues.ShouldBeEquivalentTo(expectedCompletedOrderValues,
+                $"Expected: {expectedCompletedOrderValues}" + Environment.NewLine +
+                $"Actual: {completedOrderValues}");
+        }
+        
+        /// <summary>
+        /// In this example, cash is applied first. The remaining balance can be covered in full by the LevelUp 
+        /// discount credit available.
+        /// </summary>
+        /// <remarks>
+        /// In this example, the second payment will still be responsible for some tax.
+        /// </remarks>
+        [TestMethod]
+        public void SplitTenderExample_LevelUp_Second_Discount_Remaining_Balance()
+        {
+            // $22.00 is owed, $2.00 of that is tax. $0.00 of that is tobacco/alcohol, and the customer wants to pay
+            // $12.00 towards the check all of which is discountable.
+            Check check = new Check(total: 2200, tax: 200);
+            int exemptionAmount = 0;
+            int spendAmount = 1200;
+
+            // Apply a non-LevelUp tender
+            check.ApplyTender(1000);
+
+            // Create proposed order: expected values
+            AdjustedCheckValues expectedProposedOrderValues =
+                new AdjustedCheckValues(
+                    spendAmount: 1200,
+                    taxAmount: 200,
+                    exemptionAmount: 0);
+
+            AdjustedCheckValues proposedOrderValues = ProposedOrderCalculator.CalculateCreateProposedOrderValues(
+                check.TotalOutstandingAmount,
+                check.TotalTaxAmount,
+                exemptionAmount,
+                spendAmount);
+
+            proposedOrderValues.ShouldBeEquivalentTo(expectedProposedOrderValues,
+                $"Expected: {expectedProposedOrderValues}" + Environment.NewLine +
+                $"Actual: {proposedOrderValues}");
+
+            // available discount amount $10.00
+            int availableDiscountAmount = 1000;
+
+            // POS applies $10.00 pretax discount to check and updates subtotal and tax
+            check.ApplyDiscount(availableDiscountAmount);
+
+            // Create completed order: expected values
+            AdjustedCheckValues expectedCompletedOrderValues =
+                new AdjustedCheckValues(
+                    spendAmount: 1100,      // spendAmount will include appliedDiscount and remaining tax
```

Comment:
I spent a bunch of time trying to figure out why this number is what it is.  Maybe worth adding a comment.